src/components/WebexMeeting/WebexMeeting.css

```diff
@@ -0,0 +1,15 @@
+.meeting {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+}
+
+.controls {
+  display: flex;
+  position: absolute;
+  align-self: flex-end;
+  padding-bottom: 3rem;
```

Comment:
If you're going to absolute position it, use the positioning properties instead of padding